app/Imports/ImporPenduduk.php

```diff
@@ -21,16 +21,12 @@ class ImporPenduduk implements ToCollection, WithHeadingRow, WithChunkReading, S
     protected $provinsi_id;
     protected $kabupaten_id;
     protected $kecamatan_id;
-    protected $desa_id;
-    protected $tahun;
 
     public function __construct(array $request)
```

Comment:
Boleh saya minta tolong pak, sekalian hapus fungsi constructor,
kode provinsi sampai ke kode kecamatan ambil juga dari excell biar nanti class ImporPenduduk ini bisa digunakan di pr bapak yg sinkronasi.

Ini juga harus disesuaikan di class DemoPendudukSeeder, hapus array nya.https://github.com/OpenSID/OpenDK/blob/1e7e6176543de6fa63f4dcde349a93ae31715c62/database/seeds/Demo/DemoPendudukSeeder.php#L30-L33

```suggestion
    public function __construct(array $request)
```

Reply:
> Boleh saya minta tolong pak, sekalian hapus fungsi constructor,
> kode provinsi sampai ke kode kecamatan ambil juga dari excell biar nanti class ImporPenduduk ini bisa digunakan di pr bapak yg sinkronasi.
> 
> Ini juga harus disesuaikan di class DemoPendudukSeeder, hapus array nya.
> 
> 

Terimakasih @agungsugiarto 
Awalnya kode provinsi sampai ke kode kecamatan dimasukan ke excel tapi ternyata penambahan data ini menambah juga ukuran file excel jadi supaya file yg dikirim tidak besar, tugasnya disharing : kode provinsi sampai kode kecamatan menjadi tugas OpenDK

Reply:
Baik pak saya setuju

Reply:
> Ini juga harus disesuaikan di class DemoPendudukSeeder, hapus array nya.
> 
> https://github.com/OpenSID/OpenDK/blob/1e7e6176543de6fa63f4dcde349a93ae31715c62/database/seeds/Demo/DemoPendudukSeeder.php#L30-L33

Done